test/Database/LSMTree/Model/Session.hs

```diff
@@ -603,6 +603,9 @@ openSnapshot label name = do
           Just table' ->
             newTableWith conf table'
 
+-- TODO: to match the implementation of the real table, this should not corrupt the
+--       snapshot if there are _no non-empty files_; however, since there are no such
+--       snapshots, this is probably fine
```

Comment:
Convert into a regular comment instead of a TODO. I don't think this is something we'll have to change in the future.